cmd/minishift/cmd/ssh.go

```diff
@@ -34,9 +38,13 @@ var sshCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
+
 		err := cluster.CreateSSHShell(api, args)
 		if err != nil {
-			atexit.ExitWithMessage(1, fmt.Sprintf("Cannot establish SSH connection to the VM: %s", err.Error()))
+			if err.Error() == SshCommunicationError {
```

Comment:
Are we not handling the issue twice now? Here and in _ CreateSSHShell_? Why do we need to introduce and handle _SshCommunicationError_ in the command as well, given the change in _ CreateSSHShell_?